.github/workflows/claude-review.yml

```
@@ -0,0 +1,285 @@
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v44
+        with:
+          files: |
+            client/**/*.ts
+            client/**/*.html
+            client/**/*.scss
+
+      - name: Create review context
+        id: review-context
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)